Source/Resources/MongoDB/ResourceType.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Dolittle.Runtime.ResourceTypes;
+
+namespace Dolittle.Runtime.Resources.MongoDB
+{
+    /// <summary>
+    /// Represents a <see cref="IAmAResourceType">resource type</see> for the Mongo Db resource.
+    /// </summary>
+    /// <remarks>The name is currently readModels to support for legacy cases.</remarks>
+    public class ResourceType : IAmAResourceType
+    {
+        /// <summary>
+        /// The <see cref="ResourceTypes.ResourceType"/> name.
+        /// </summary>
+        public static ResourceTypes.ResourceType ResourceTypeName => "readModels";
```

Comment:
Let's make a remark that this is called "readModels" and not something with MongoDB to be backwards compatible with the old style of resource configuration and use the old databases. Also - let's not this in the release notes so we don't get confused in the future.

Reply:
Done